### PR TITLE
feat(distribution): allow displaying more information in the answer

### PR DESCRIPTION
**Pull request by anehx**
Screenshot from 2022-04-25 16-22-34

### CODE REVIEW

File: packages/distribution/addon/components/cd-inquiry-dialog/inquiry-part.hbs [hunk 1]
```diff
@@ -43,6 +43,35 @@
...
+    class="uk-margin-remove"
+    @text={{this.requestInfo}}
+    @length={{500}}
+  />
+{{else if this.answerInfo}}
+  {{#each this.answerInfo as |info index|}}
+    <div
+      class={{if
+        (lt index (sub this.answerInfo.length 1))
+        (if this.config.ui.small "uk-margin-small" "uk-margin")
```
[the remark this answers is not in the document]
Comment (anehx) [reply packages/distribution/addon/components/cd-inquiry-dialog/inquiry-part.hbs]: Didn't know that one, nice!

File: packages/distribution/addon/components/cd-truncated.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+export default class CdTruncatedComponent extends Component {
+  @tracked expand = false;
+
+  get length() {
+    return parseInt(this.args.length);
+  }
+
+  get displayedText() {
+    if (this.truncate && !this.expand) {
+      return `${this.args.text.substring(0, this.args.length - 3).trim()}...`;
```
[the remark this answers is not in the document]
Comment (anehx) [reply packages/distribution/addon/components/cd-truncated.js]: Added a comment

[the remark this answers is not in the document]
Comment (anehx) [reply packages/distribution/addon/components/cd-truncated.js]: Fixed

**Comment by anehx** [comment]
> The more / less links are a bit difficult to see, especially when there is a lot of text. Uikit doesn't offer anything in this direction, besides like an overlay component, correct?
> 
> image

I don't think this styling would help. This would clutter the UI and it's a pattern that is used for articles - not app UIs.. Let's keep it like it is and gather some experience instead of putting too many thoughts into it already.

**Review by luytena — APPROVED** [review]
:100:

**Pull request merged**